tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -92,4 +92,18 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, wantOutput)
 		})
 	})
+
+	Context("When executing catalog list components with registry that is not set up properly", func() {
```

Comment:
Can you also check for the warning message that's logged?

Reply:
Yeah, that's a good question, I tried that before but failed. From CLI perspective we can see the warning message and I think both warning and regular messages are in `stdout`, but from testing perspective inside `CmdShouldPass` function, I find it wraps `ginkgo` and `gomega` functions for returning command output and warning message won't be returned (only regular message is returned), which means we can't catch the warning messages from test cases, I suspect in test cases warning message redirects to other stream, that's why I use this way for testing (to test invalid registry won't block listing valid registry).